doc/HOWTO_RELEASE.rst.txt

```diff
@@ -40,27 +40,27 @@ Release Scripts
 
 Supported platforms and versions
 ================================
-Python 2.7 and >=3.4 are the currently supported versions when building from
-source.  We test NumPy against all these versions every time we merge code to
-master.  Binary installers may be available for a subset of these versions (see
-below).
+`NEP 29`<https://numpy.org/neps/nep-0029-deprecation_policy.html>_ outlines
```

Comment:
reST formatting should be:
```
`NEP 29 <https://numpy.org/neps/nep-0029-deprecation_policy.html>`__
```

Reply:
Thanks. I was waiting for circleci to verify the artifact but this saves one cycle.